packages/forms/src/form_builder.ts

```diff
@@ -15,11 +15,18 @@ import {FormArray, UntypedFormArray} from './model/form_array';
 import {FormControl, FormControlOptions, FormControlState, UntypedFormControl} from './model/form_control';
 import {FormGroup, UntypedFormGroup} from './model/form_group';
 
-function isAbstractControlOptions(options: AbstractControlOptions|
-                                  {[key: string]: any}): options is AbstractControlOptions {
-  return (<AbstractControlOptions>options).asyncValidators !== undefined ||
-      (<AbstractControlOptions>options).validators !== undefined ||
-      (<AbstractControlOptions>options).updateOn !== undefined;
+function isAbstractControlOptions(options: AbstractControlOptions|{[key: string]: any}|null|
+                                  undefined): options is AbstractControlOptions {
+  return !!options &&
+      (options.asyncValidators !== undefined || options.validators !== undefined ||
+       options.updateOn !== undefined);
+}
+
+function isFormControlOptions(options: FormControlOptions|{[key: string]: any}|null|
+                              undefined): options is FormControlOptions {
+  return !!options &&
+      (options.asyncValidators !== undefined || options.validators !== undefined ||
+       options.updateOn !== undefined || options.initialValueIsDefault !== undefined);
```

Comment:
Can we reuse the `isAbstractControlOptions` here?
```suggestion
  return !!options && (isAbstractControlOptions(options) || options.initialValueIsDefault !== undefined);
```

Reply:
Done.